community/neo4j/src/test/java/schema/MultipleIndexPopulationStressIT.java

```diff
@@ -398,4 +374,36 @@ public Estimates calculateEstimates( ToIntFunction<Value[]> valueSizeCalculator
             return knownEstimates( count, 0, count * TOKENS.length / 2, 0, count * TOKENS.length / 2 * Long.BYTES, 0, 0 );
         }
     }
+
+    private RandomValues.Configuration getConfiguration()
+    {
+        return new RandomValues.Configuration()
```

Comment:
Same as above about duplicate very similar config. Please let RandomValues and RandomRule be friends, please?

Reply:
We could just move `RandomRule` into this new package and it's 3 neo4j related imports (which will mostly be removed any way).